finance_manager/src/main/resources/application.yml

```diff
@@ -24,7 +24,7 @@ spring:
       hibernate:
         dialect: org.hibernate.dialect.PostgreSQLDialect
     hibernate:
-      ddl-auto: validate
+      ddl-auto: create
```

Comment:
as per Vitaliy must be none or validate. 
DB changes should be via migrations. Was it for the test purposes?

Reply:
Yes, for tests, this pull request is not ready for merging due to unimplemented functions. will be fixed to validate in the final version